tests/cli/test_rasa_data.py

```diff
@@ -202,6 +202,30 @@ def test_validate_files_form_not_found_invalid_domain(
         data.validate_files(namedtuple("Args", args.keys())(*args.values()))
 
 
+def test_validate_files_with_active_loop_null(tmp_path: Path):
```

Comment:
I tried running the test locally and it failed. 
It fails because of this line: https://github.com/RasaHQ/rasa/blob/02fa04c358e7c8d19a970c6c5d4fbbc0c2e7d2ee/rasa/validator.py#L436
Returns false which results in an invalid domain and thus an error.
However, I see that the test is passed on CI. Any ideas about this?

Reply:
Hm I was able to run it locally without issues. The only reason why it can be is when `data/test_restaurantbot/domain.yml`, you sure it's not the case?